includes.container/etc/xdg/swayrbar/config.toml

```diff
@@ -0,0 +1,68 @@
+refresh_interval = 1000
+
+[[modules]]
+name = "window"
+instance = "0"
+format = "🪟 {title} — {app_name}"
+html_escape = false
+
+[modules.on_click]
+Left = [
+    "swayr",
+    "switch-to-urgent-or-lru-window",
+]
+Right = [
+    "kill",
+    "{pid}",
+]
+
+[[modules]]
+name = "sysinfo"
+instance = "0"
+format = "💻 CPU: {cpu_usage:{:5.1}}% Mem: {mem_usage:{:5.1}}% Load: {load_avg_1:{:5.2}} / {load_avg_5:{:5.2}} / {load_avg_15:{:5.2}}"
+html_escape = false
+
+[modules.on_click]
+Left = [
+    "foot",
+    "htop",
+]
+
+[[modules]]
+name = "battery"
+instance = "0"
+format = "🔋 Bat: {state_of_charge:{:5.1}}%, {state}, Health: {state_of_health:{:5.1}}%"
+html_escape = false
+
+[[modules]]
+name = "pactl"
+instance = "0"
+format = "🔈 Vol: {volume:{:3}}%{muted}"
+html_escape = true
+
+[modules.on_click]
+Right = [
+    "pactl",
```

Comment:
this program is not installed

```console
[host-shell]$ journalctl --user --identifier=sway
Aug 31 00:15:36 tux sway[19153]: [2024-08-30T22:15:36Z ERROR swayrbar::module::pactl] Could not run pactl: No such file or directory (os error 2)
Aug 31 00:15:36 tux sway[19153]: [2024-08-30T22:15:36Z ERROR swayrbar::module::pactl] Could not run pactl: No such file or directory (os error 2)

```

Reply:
Installing `pactl` works although there is no pulseaudio (pipewire is used by debian).

Using `wpctl` would be more intuitiv and does not need additonal programs but it is not supported yet by swayrbar.